resources/profiles/Anker.ini

```diff
@@ -186,87 +847,87 @@ compatible_printers_condition = printer_model=~/(M5).*/ and nozzle_diameter[0]==
 # So having some leeway to get good bed adhesion is not a luxury for many users
 
 [filament:*common*]
-cooling = 0
-compatible_printers = 
-extrusion_multiplier = 1
-filament_cost = 0
-filament_density = 0
-filament_diameter = 1.75
-filament_notes = ""
-filament_settings_id = ""
-filament_soluble = 0
-min_print_speed = 30
-slowdown_below_layer_time = 8
-compatible_printers_condition = printer_notes=~/.*PRINTER_VENDOR_ANKERMAKE.*/
+    cooling = 1
```

Comment:
I would set cooling to 0 and overwrite where it needs to be 1, or just set to 0 for ABS